pymtl3/stdlib/ifcs/SendRecvIfc.py

```diff
@@ -28,7 +28,13 @@ def construct( s, Type ):
     s.MsgType = Type
 
   def line_trace( s ):
-    return enrdy_to_str( s.msg, s.en, s.rdy )
+    try:
+      trace_len = s.trace_len
+    except AttributeError:
+      s.trace_len = len( "{}".format( s.MsgType() ) )
```

Comment:
this is correct -- it will throw exception for the first time and then always get the s.trace_len